include/image-commands.mk

```diff
@@ -298,3 +298,11 @@ define Build/kernel2minor
 	kernel2minor -k $@ -r $@.new $(1)
 	mv $@.new $@
 endef
+
+define Build/sdcard-image
+	if [ -n "$(UBOOT_IMAGE)" ]; then UBOOT_IMAGE="$(STAGING_DIR_IMAGE)/$(UBOOT_IMAGE)"; fi; \
+	$(SCRIPT_DIR)/gen_sdcard_image.sh "$@" "$@.boot" "$(IMAGE_ROOTFS)" \
```

Comment:
The command expects bootfs image to have `.boot` suffix, but definition in:
https://github.com/openwrt/openwrt/blob/dc629d9cf5afc03f67a294078ca1001e831c66c9/target/linux/mvebu/image/Makefile#L43
creates bootfs image with `.bootimg` suffix. That produces sdcard image with empty boot partition.

Reply:
good catch ; i think renaming `$@.bootimg` => `$@.boot` should be a reasonable solution for now